iohub/mmstack.py

```diff
@@ -140,6 +141,8 @@ def _parse_data(self):
                 "smaller than the number of channels. "
                 f"Completing with fallback names: {self.channel_names}."
             )
+        # not all positions in the position list may have been acquired
+        xarr = xarr[: self.positions]
```

Comment:
Will the missing positions always be at the end of the list?

Reply:
Just checked. Yes, this is the case, at least for now, due to a MM bug:
https://github.com/micro-manager/micro-manager/issues/1723
https://github.com/czbiohub-sf/iohub/pull/197